src/constants/entities.js

```diff
@@ -237,6 +237,15 @@ const sector = {
   children: [blackHole.key, note.key, system.key],
 };
 
+// that is not realy an entity
+// this entry is here to display the settings in the entity sidebar
+const settings = {
```

Comment:
I do the same for navigation... should build a way to extract these two but for now this is fine. Also, misspelling in your comment above.

Reply:
One "problem" here is, that there is a empty settings object in the exported JSON.

Reply:
Yeah, that's not ideal. There's an empty navigation object as well. I think that's alright for now until I figure out a good way of splitting actual entities from objects that configure pages.